specification/purview/data-plane/Azure.Analytics.Purview.PDS/preview/2022-11-01-preview/pds.json

```diff
@@ -0,0 +1,463 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview Distribution Service REST API Document",
+    "description": "Purview distribution service enables various Azure RPs to fetch the policies and attributes for data governance.",
+    "version": "2022-11-01-preview"
+  },
+  "paths": {
+    "/datasource-preferences": {
+      "get": {
+        "tags": [
+          "DatasourcePreferences"
+        ],
+        "description": "Get Datasource Preferences for a given scope.",
+        "operationId": "DatasourcePreferences_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "scope",
+            "description": "Scope.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "isProjectGondor",
+            "description": "Is call from a Gondor client.",
+            "type": "boolean"
+          },
```

Comment:
What is this?  And how does this affect the behavior of the request?

Reply:
HI Mike, right now this API is supporting two Purview account types, one for Purview Babylon (First purview offering) and another for newer Purview Gondor. This parameter is added for supporting backward compatibility among these two new versions.
We will not have this parameter in the GA APIs as all the Babylon Accounts will be migrated to Gondor accounts sooner or later.
For preview version this parameter is important to keep.

Reply:
I am resolving this conversation based on above explanation. Let me know if this needs further clarification.